sharepoint-adfs/README.md

```diff
@@ -0,0 +1,95 @@
+# AzureRM template for SharePoint 2016 and 2013 configured with ADFS
+
+## Description
+
+This template deploys 3 new Azure VMs, each with its own public IP address and subnet:
+
+* A new AD Domain Controller with a root certificate authority (AD CS) and AD FS configured
+* A SQL Server 2016
+* A single server running a SharePoint 2016 or 2013 farm configured with 1 web application and 2 zones. Default zone is using Windows authentication and Intranet zone is using federated authentication with ADFS. Latest version of claims provider [LDAPCP](http://ldapcp.com/) is installed and configured. Some service applications and site collections are also provisionned.
+
```

Comment:
I like your comment that this will take 1.5 hours to deploy - that would be a good addition to the readme, you can be hand wavy (e.g. this will take 1-2 hours to deploy)...

Reply:
Indeed, it's a very good suggestion! Done